README.md

```diff
@@ -10,12 +10,60 @@ Usage
 ```java
 Duktape duktape = Duktape.create();
 try {
-  Log.d("Greeting", duktape.evaluate("'hello world'.toUpperCase();"));
+    Log.d("Greeting", duktape.evaluate("'hello world'.toUpperCase();"));
 } finally {
-  duktape.close();
+    duktape.close();
 }
 ```
 
+## Calling Java from Javascript
+
+You can bind a Java object to a Javascript global, and call Java functions from Javascript!
+Currently, the following Java types are supported for function arguments and return values:
+
+ * `boolean` and `Boolean`
+ * `int` and `Integer`
+ * `double` and `Double`
+ * `String`
+
+`void` return values are also supported.
+
+### Example
+
+First, define a Java interface that declares the methods you would like to call from Javascript:
+
+```java
+interface PersonInterface {
+    String getName();
+}
+```
+
+Next, implement the interface in Java code:
+
+```java
+class Person implements PersonInterface {
```

Comment:
Nit: is there a more plausible example? Something to get people’s minds racing at the possibilities? 
One fun option: rather than returning a constant string, you could prompt for one with `java.io.Console` ?